### PR TITLE
No show option and new overlay

### PR DESCRIPTION
**Pull request by hli-ng**

### CODE REVIEW

File: examples/volume-viewer-demo.js [hunk 1]
```diff
@@ -98,6 +99,7 @@ $(function() {
         viewer.loadVolumes({
           volumes: [
             {
+              show_volume: false,
```
Comment (rnemec-ng) [inline examples/volume-viewer-demo.js]: I think you may be able to achive the same by not setting the `element_id` or `viewer_insert_class` or missing the whole `template: {...}` section... Worth trying at least...

File: src/brainbrowser/volume-viewer/modules/loading.js [hunk 2]
```diff
@@ -508,7 +509,7 @@ BrainBrowser.VolumeViewer.modules.loading = function(viewer) {
       }
 
       Array.prototype.forEach.call(template, function(node) {
-        if (node.nodeType === 1) {
+        if (node.nodeType === 1 && volume_description.show_volume !== false) {
```
Comment (rnemec-ng) [inline src/brainbrowser/volume-viewer/modules/loading.js]: 10 lines up is `if` skipping this when no `template` is provided (note that `template_options` defaults to `{}`).

File: examples/volume-viewer-demo.js (hunk 1 above)
Comment (hli-ng) [reply examples/volume-viewer-demo.js]: our current client implementation doesn't provide template: {...} at all and the volume viewer still show.

File: src/brainbrowser/volume-viewer/modules/loading.js (hunk 2 above)
Comment (hli-ng) [reply src/brainbrowser/volume-viewer/modules/loading.js]: see reply to previous comment

**Pull request merged**